PowerEditor/src/Notepad_plus.cpp

```diff
@@ -230,9 +234,12 @@ LRESULT Notepad_plus::init(HWND hwnd)
     const ScintillaViewParams & svp1 = nppParam.getSVP();
 
 	int tabBarStatus = nppGUI._tabStatus;
+	
 	_toReduceTabBar = ((tabBarStatus & TAB_REDUCE) != 0);
-	int iconDpiDynamicalSize = nppParam._dpiManager.scaleY(_toReduceTabBar?13:20);
-	_docTabIconList.create(iconDpiDynamicalSize, _pPublicInterface->getHinst(), docTabIconIDs, sizeof(docTabIconIDs)/sizeof(int));
+	int iconDpiDynamicalSize = nppParam._dpiManager.scaleY(_toReduceTabBar ? 12 : 18);

```

Comment:
Is any reason for changing the scale size?

Reply:
@donho I changed it because of discoveries we discussed here: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8068#issuecomment-608635130